test-zone-data/Consistency-TP/consistency06/README.md

```diff
@@ -0,0 +1,213 @@
+# CONSISTENCY05
+
+[This directory](.), i.e. the same directory as this README file, holds zonefiles and `coredns` configuration files for scenarios for test case CONSISTENCY06:
+
+* ONE-SOA-MNAME-1
+* ONE-SOA-MNAME-2
+* ONE-SOA-MNAME-3
+* ONE-SOA-MNAME-4
+* MULTIPLE-SOA-MNAMES-1
+* MULTIPLE-SOA-MNAMES-2
+* NO-RESPONSE
+
+## zonemaster-cli commands and their output for each test scenario
+
+The level (`--level`) must be set to the lowest level of the message tags. For this test case `INFO` is the lowest level.
+
+Scenario name         | Mandatory message tags                            | Forbidden mess
+age tags
```

Comment:
There is a line break at an unfortunate place, appearing multiple times in this document:
```suggestion
Scenario name         | Mandatory message tags                            | Forbidden message tags
```